rebar.config

```diff
@@ -27,6 +27,7 @@
     {elli, "3.3.0"},
     {bbmustache, {git, "https://github.com/helium/bbmustache", {branch, "data-fun"}}},
     {iso8601, {git, "https://github.com/erlsci/iso8601", {branch, "master"}}},
+    {chatterbox, {git, "https://github.com/tsloughter/chatterbox", {branch, "master"}}},
```

Comment:
Going to upstream chatterbox for grpcbox improved skf list performance by more than 10x.
If a Router has a significant amount of devices, this can have a huge impact. 
Listing SKFs at the start of a reconcile can take up to 2 minutes for a Router with more than 75k devices under best conditions.

Reply:
Will discuss next week.